addons/binding/org.openhab.binding.miio/README.md

```diff
@@ -0,0 +1,54 @@
+# Xiaomi Robot Vacuum Binding
+
+This Binding is used to control Xiaomi Mi IO devices.
+
+![MIIO logo](doc/miio.png)
+
+## Supported Things
+
+This Binding supports Xiaomi Mi IO devices (Several Xiaomi wifi devices).
+Currently mainly the Xiaomi Robot Vacuum is supported, remaining devices are experimental.
+Experimental means that most likely you can send commands to the devices
+
+The following things are available
+
+| Thing Type | Description |
+|--------|---------| 
+| miio:generic | Generic type for discovered devices. Once the tolken is available and the device model is determined, this thingtype will automatically change to the appropriate thingtype for the model |
+| miio:vacuum | For Xiaomi Robot Vacuum |
```

Comment:
Consider leaving out the "For". Also I wonder if a Version / tested firmware version would be good added info here.